pmd-java/src/main/java/net/sourceforge/pmd/lang/java/types/TestingFrameworkTypeUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.types;
+
+import net.sourceforge.pmd.lang.java.ast.ASTMethodDeclaration;
+
+/**
+ *  Public Utilities to check the type of testing framework
+ */
+public final class TestingFrameworkTypeUtil {
```

Comment:
Instead of creating a new public API class, we should either move it into the package `net.sourceforge.pmd.lang.java.rule.internal` or add the methods to `net.sourceforge.pmd.lang.java.rule.internal.JavaRuleUtil`. This way, to keep our public facing API small (and we can still change the implementation without worrying about compatibility).